model/custom_field_v2.go

```diff
@@ -0,0 +1,55 @@
+package model
+
+import "github.com/incident-io/singer-tap/client"
+
+type customFieldV2 struct{}
+
+var CustomFieldV2 customFieldV2
+
+func (customFieldV2) Schema() Property {
+	return Property{
+		Types: []string{"object"},
+		Properties: map[string]Property{
+			"id": {
+				Types: []string{"string"},
+			},
+			"catalog_type_id": {
```

Comment:
This suggests that we should put catalog _types_ into the warehouse, at least (or remove this property)

Reply:
I've put a big "maybe" on the catalog API and was going to leave it off for now - agree would be good to hide these for now.